facades/PC/src/main/java/org/terasology/engine/Terasology.java

```diff
@@ -346,8 +346,9 @@ private static int getLastNumber(String str) {
      * <p>
      * On Windows, you may be able to set a limit using one of these external tools:
      * <ul>
-     *     <li><a href="https://docs.microsoft.com/en-us/windows-hardware/drivers/devtest/application-verifier">Application Verifier
-     *         (<code>AppVerif.exe</code>)</a>, available from the Windows SDK</li>
+     *     <li>
+     *     <a href="https://docs.microsoft.com/en-us/windows-hardware/drivers/devtest/application-verifier">Application Verifier (<code>AppVerif.exe</code>)</a>,
```

Comment:
any clue about getting rid of this LineLenghtCheck. sometimes its better to format this with one long line. 

I think this is a lot more readable then trying to work out how to chop the line to the right length. 
```
     * <ul>
     *     <li><a href="https://docs.microsoft.com/en-us/windows-hardware/drivers/devtest/application-verifier">Application Verifier (<code>AppVerif.exe</code>)</a>,available from the Windows SDK</li>
     *     <li><a href="https://github.com/lowleveldesign/process-governor">Process Governor (<code>procgov</code>)</a>,an open source third-party tool</li>
     * </ul>
```

Reply:
As @skaldarnar mentioned on Discord, for these "sometimes" cases there's annotations that allow exactly that: ignoring the check rule in a particular case.